### PR TITLE
Replace namespace celmath with celestia::math

### PR DESCRIPTION
**Pull request by 375gnu**

### CODE REVIEW

File: src/celengine/deepskyobj.cpp [hunk 1]
```diff
@@ -75,12 +76,13 @@ bool DeepSkyObject::pick(const Eigen::ParametrizedLine<double, 3>& ray,
                          double& cosAngleToBoundCenter) const
 {
     if (isVisible())
-        return celmath::testIntersection(ray,
-                                         celmath::Sphered(position, static_cast<double>(radius)),
+    {
+        return math::testIntersection(ray,
+                                         math::Sphered(position, static_cast<double>(radius)),
```
Comment (ajtribick) [inline src/celengine/deepskyobj.cpp:81]: Need to adjust indentation here

File: src/celengine/dsorenderer.h [hunk 2]
```diff
@@ -27,7 +27,7 @@ class DSORenderer : public ObjectRenderer<DeepSkyObject*, double>
 
     Eigen::Vector3d     obsPos;
     Eigen::Matrix3f     orientationMatrixT;
-    celmath::Frustum    frustum         { celmath::degToRad(celestia::engine::standardFOV), 1.0f, 1.0f };
+    celestia::math::Frustum    frustum         { celestia::math::degToRad(celestia::engine::standardFOV), 1.0f, 1.0f };
```
Comment (ajtribick) [inline src/celengine/dsorenderer.h:30]: spacing should be fixed here

File: src/celengine/galaxy.cpp [hunk 3]
```diff
@@ -123,10 +124,10 @@ bool Galaxy::pick(const Eigen::ParametrizedLine<double, 3>& ray,
                                   getRadius()*(galacticForm->scale.z() + kRadiusCorrection));
     Eigen::Matrix3d rotation = getOrientation().cast<double>().toRotationMatrix();
 
-    return celmath::testIntersection(
-        celmath::transformRay(Eigen::ParametrizedLine<double, 3>(ray.origin() - getPosition(), ray.direction()),
+    return math::testIntersection(
+        math::transformRay(Eigen::ParametrizedLine<double, 3>(ray.origin() - getPosition(), ray.direction()),
```
Comment (ajtribick) [inline src/celengine/galaxy.cpp:128]: Adjust indentation

File: src/celengine/globular.cpp [hunk 4]
```diff
@@ -86,9 +87,9 @@ bool Globular::pick(const Eigen::ParametrizedLine<double, 3>& ray,
      * when they are drawn.
      */
     Eigen::Vector3d p = getPosition();
-    return celmath::testIntersection(celmath::transformRay(Eigen::ParametrizedLine<double, 3>(ray.origin() - p, ray.direction()),
+    return math::testIntersection(math::transformRay(Eigen::ParametrizedLine<double, 3>(ray.origin() - p, ray.direction()),
                                                            getOrientation().cast<double>().toRotationMatrix()),
-                                     celmath::Sphered(getRadius() * (1.0f + kRadiusCorrection)),
+                                     math::Sphered(getRadius() * (1.0f + kRadiusCorrection)),
```
Comment (ajtribick) [inline src/celengine/globular.cpp:92]: indentation

File: src/celestia/win32/winstarbrowser.cpp [hunk 5]
```diff
@@ -308,12 +309,12 @@ int CALLBACK StarBrowserCompareFunc(LPARAM lParam0, LPARAM lParam1,
                 d0 = sortInfo->ucPos.offsetFromLy(star0->getPosition()).norm();
             if (d1 < 1.0f)
                 d1 = sortInfo->ucPos.offsetFromLy(star1->getPosition()).norm();
-            return (int) celmath::sign(star0->getApparentMagnitude(d0) -
+            return (int) math::sign(star0->getApparentMagnitude(d0) -
```
Comment (ajtribick) [inline src/celestia/win32/winstarbrowser.cpp:312]: Indentation needs adjusting

**Review by ajtribick — CHANGES_REQUESTED** [review]

File: src/celengine/deepskyobj.cpp [hunk 6]
```diff
@@ -75,12 +76,13 @@ bool DeepSkyObject::pick(const Eigen::ParametrizedLine<double, 3>& ray,
...
                                          distanceToPicker,
                                          cosAngleToBoundCenter);
```
Comment (375gnu) [inline src/celengine/deepskyobj.cpp:83]:
```suggestion
                                      math::Sphered(position, static_cast<double>(radius)),
                                      distanceToPicker,
                                      cosAngleToBoundCenter);
```

File: src/celengine/dsorenderer.h (hunk 2 above)
Comment (375gnu) [inline src/celengine/dsorenderer.h:30]:
```suggestion
    celestia::math::Frustum frustum{ celestia::math::degToRad(celestia::engine::standardFOV), 1.0f, 1.0f };
```

File: src/celengine/dsorenderer.h [hunk 7]
```diff
@@ -27,7 +27,7 @@ class DSORenderer : public ObjectRenderer<DeepSkyObject*, double>
...
     DSODatabase*        dsoDB           { nullptr };
```
Comment (375gnu) [inline src/celengine/dsorenderer.h:31]:
```suggestion
    celestia::math::Frustum frustum    { celestia::math::degToRad(celestia::engine::standardFOV), 1.0f, 1.0f };
    DSODatabase*        dsoDB           { nullptr };
```

**Review by ajtribick — APPROVED** [review]

**Pull request merged**